docker-compose.yml

```diff
@@ -4,11 +4,13 @@ services:
     image: postgres
     #restart: always
     environment:
+      POSTGRERS_USER: postgres
       POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: travis-ci-test
     expose:
-      - "5432"
+      - "5433"
```

Comment:
This shouldn't need to be changed. Were you getting a conflict? If so, that indicates another service listening, which we should not be seeing.